paymentsheet/src/test/java/com/stripe/android/link/LinkControllerViewModelTest.kt

```diff
@@ -184,49 +244,18 @@ class LinkControllerViewModelTest {
         }
     }
 
-    @Test
-    fun `onCreatePaymentMethod() fails when account is not set`() = runTest {
```

Comment:
👀 Deleted this test since the call is done through `LinkAccountManager` now which owns this logic.